src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, number) {
+      operation(number);
+
+      return calculator;
+    },
+    add(n) {
+      calculator.result += n;
+    },
```

Comment:
It would be better to return result and save him in operate method.
```suggestion
    add(a, b) {
      return a + b;
    },
```